docs/theming/developers.md

```diff
@@ -341,24 +350,24 @@ and applies values to the theming tokens, depending on the context received.
 ```
 
 For more information on the significance of the context values (i.e.
-`ColorTheme`), read "[Background][backgrounds]".
+`ColorTheme`), read “[Background][backgrounds]”.
 
 ### Future state - style queries  <rh-tag color="purple" icon="notification-fill">Planned</rh-tag>
 
-In a not so distant future, we will be able to replace the context protocol
+In the not-so-distant future, we will be able to replace the context protocol
 completely and remove this code by implementing the web standard [container
 style queries][stylequeries].
 
 In anticipation of this upcoming browser feature, we attempt to ensure that our
-theming system as implemented today using context can easily replaced with style
-queries in the near future.
+theming system as implemented today using context can easily be replaced with
+style queries in the near future.
 
-<rh-alert>Note, the examples in this section are hypothetical, the final markup
-and styles may not be the same</rh-alert>
+<rh-alert>Note: the examples in this section are hypothetical, the final markup
```

Comment:
`hypothetical, the` → `hypothetical. The`

Reply:
[8ae3170](https://github.com/RedHat-UX/red-hat-design-system/pull/1937/commits/8ae31707ea11f29ae967709bb95afebae9820f44)